workflow-runs.js

```diff
@@ -20,11 +20,11 @@ const waitForWorkflowRunToFinish = async (context, token, owner, repo, workflowR
           return res
       }
       if (context.log) context.log(`Waiting for workflow run ${workflowRunId} (current status: ${res.status})`)
-      if (counter++ > 30) throw new Error(`Times out waiting for workflow run ${workflowRunId}?`)
-      await sleep(1000)
+      if (counter++ > 60) throw new Error(`Times out waiting for workflow run ${workflowRunId}?`)
```

Comment:
`s/Times/Timed/` ?

Reply:
Done: https://github.com/git-for-windows/git-for-windows-automation/pull/57/commits/684f4d683b4692d1829db344d4a29c5ae300036a